Dragonflight/ShamanElemental.lua

```diff
@@ -238,7 +262,7 @@ spec:RegisterAuras( {
     flame_shock = {
         id = 188389,
         duration = function () return level > 58 and fire_elemental.up and 36 or 18 end,
-        tick_time = function () return 2 * haste end,
+        tick_time = function () return 2 * haste * (talent.flames_of_the_cauldron.enabled and 0.85 or 1) end,
```

Comment:
Parentheses are spaced out in addon code (based on my personal preference).

`( talent.flames_of_the_cauldron.enabled and 0.85 or 1 )`